### DISCUSSION TITLE
Comentários sobre o fórum

### DISCUSSION THREAD

**Discussion by wandersoncferreira** [clj-br/forum#11] [Geral]
Gostaria de ouvir a opinião de vocês sobre a estrutura do Github Discussions e se acham que é viável seguir por aqui com esse projeto.

**Comment by pfeodrippe** [comment]
Ótima iniciativa, Wanderson, Eric e Lucas!

Vocês sabem se as discussões aqui são indexadas pelo Google? E se dá pra gerar um arquivo com as mensagens e imagens das discussões daqui?

Vai ser massa demais pra tentar crescer e manter o Clojure BR :)

**Reply by wandersoncferreira** [reply]
> Vocês sabem se as discussões aqui são indexadas pelo Google?

Não achei uma menção direta sobre isso, mas parece que sim. Eu consegui achar via google algumas coisas do Babashka discussions. No processo acabei encontrando alguns links dessa comunidade aqui que parece bem grande e ativa https://github.com/google/or-tools/discussions

**Reply by pfeodrippe** [reply]
Boa!! Valeu

**Reply by wandersoncferreira** [reply]
> E se dá pra gerar um arquivo com as mensagens e imagens das discussões daqui?

Parece que não dá :/ https://github.com/github/feedback/discussions/3315
